setup.py

```diff
@@ -56,6 +56,11 @@
     python_requires=">=3.7",
     platforms="Linux, Mac OS X, Windows",
     keywords=["Interactive", "Interpreter", "Shell", "Web"],
+    project_urls={
+        "Documentation": "http://nbformat.readthedocs.io/",
+        "Source": "https://github.com/jupyter/nbformat",
```

Comment:
Can you please add funding here as well?

Reply:
Done :+1: